content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -136,16 +136,7 @@ Introspecting based on datasource defined in prisma/schema.prisma …
 Run prisma generate to generate Prisma Client.
 ```
 
-## Using environment variables
-
-You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
-
-- Keep secrets out of the schema file
-- Improve portability of the schema file
```

Comment:
I think these 3 lines were actually useful here @molebox - they frame why you would even need to use environment variables in the schema file itself. The `to provide configuration options when a CLI command is invoked` half sentence of course was wrong/to limited, but if you remove that you can include these three lines again.